wordlist.txt

```diff
@@ -4,6 +4,18 @@
 /phpinfo.php
 /info.php
 /phptest.php
+/etc/passwd
+/../etc/passwd
+/../../etc/passwd
+/../../../etc/passwd
+/../../../../etc/passwd
+/../../../../../etc/passwd
+/../../../../../../etc/passwd
+/../../../../../../../etc/passwd
+/../../../../../../../../etc/passwd
+/../../../../../../../../../etc/passwd
+/../../../../../../../../../../etc/passwd
+/phppgadmin
```

Comment:
I might have to add the `-L` flag in meg for this path, since `/phppgadmin` might redirect to `/phppgadmin/`.